samples/modules/mctp/mctp_endpoint/src/main.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include <unistd.h>
+#include <zephyr/kernel.h>
+#include <zephyr/types.h>
+#include <zephyr/mctp/mctp_uart.h>
+#include <libmctp.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(mctp_endpoint);
+
+#include <zephyr/drivers/uart.h>
+
+static struct mctp *mctp_ctx;
+
+#define LOCAL_HELLO_EID 10
+
+#define REMOTE_HELLO_EID 20
+
+K_SEM_DEFINE(mctp_rx, 0, 1);
+
+static void rx_message(uint8_t eid, bool tag_owner, uint8_t msg_tag, void *data, void *msg,
+		       size_t len)
+{
+	switch (eid) {
+	case REMOTE_HELLO_EID:
+		LOG_INF("got mctp message %s for eid %d, replying to 5 with \"world\"", (char *)msg,
```

Comment:
What does "replying to 5 with" mean here?

Reply:
The endpoint it’s assuming it’s replying to, I can adjust the message string here

Reply:
The endpoint it’s replying to, though I get that’s not clear, and not even known so I’ll correct this